src/Footer/Footer.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styles from './Footer.scss';
+
+function Footer() {
```

Comment:
Let's use function expression for stateless components: `const Footer = () => (<div></div>)`